packages/app-store/_components/AppCard.tsx

```diff
@@ -16,24 +16,22 @@ export default function AppCard({
   switchOnClick,
   switchChecked,
   children,
-  setAppData,
   returnTo,
   teamId,
-  disableSwitch,
-  LockedIcon,
+  appDataSchema,
```

Comment:
We are using `appDataSchema` for it's type only. So, I don't think we should pass it as prop.
Also, I don't think we need to pass in the schema type to `useAppContextWithSchema`, we can work without that.

Reply:
Fixed it.